examples/gno.land/r/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,253 @@
+package gnofundme
+
+import (
+	"bytes"
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	pkg "gno.land/p/demo/gnofundme"
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	campaigns      *avl.Tree
+	users          *avl.Tree
+	campaignPrefix = "/r/demo/gnofundme:"
+)
+
+func init() {
+	campaigns = avl.NewTree()
+	users = avl.NewTree()
+}
+
+func NewUser(name string, email string) string {
```

Comment:
What happen if the variables are empty? Also you can change the input part from
`(name string, email string)` to `(name, email string)` 
since both are strings (but that's a matter of taste)

Reply:
Same thing here for verification, the other issue is as we're talking about web3, a user is supposed to be identified only by his address, but I may use profile pkg, who is a work in progress, to identify people. I'll need a bunch of people's opinions to take decision on how to handle this.